tachyon/math/polynomials/univariate/two_adic_subgroup.h

```diff
@@ -46,26 +47,34 @@ class TwoAdicSubgroup {
 
   // Compute the low-degree extension of each column in |mat| onto a coset of
   // a larger subgroup.
-  virtual void CosetLDEBatch(RowMajorMatrix<F>& mat, size_t added_bits,
-                             F shift) {
+  virtual void CosetLDEBatch(Eigen::MatrixBase<RowMajorMatrix<F>>& mat,
+                             size_t added_bits, F shift) {
     if constexpr (F::Config::kModulusBits > 32) {
       NOTREACHED();
     }
     IFFTBatch(mat);
     Eigen::Index rows = mat.rows();
     Eigen::Index cols = mat.cols();
+    Eigen::Index new_rows = rows << added_bits;
 
     // Possible crash if the new resized length overflows
-    mat.conservativeResizeLike(
-        RowMajorMatrix<F>::Zero(rows << added_bits, cols));
+    RowMajorMatrix<F> new_mat(new_rows, cols);
+    OMP_PARALLEL_FOR(Eigen::Index row = 0; row < new_rows; ++row) {
+      if (row < rows) {
+        new_mat.row(row) = mat.row(row);
+      } else {
+        new_mat.row(row).setZero();
+      }
+    }
+    mat = std::move(new_mat);
```

Comment:
shouldn't this be a separate commit like how https://github.com/kroma-network/tachyon/pull/521/commits/95a5d8886b963b5d0f14b7e383fa6a16556a97cd is?

Reply:
Looks like kind of mixed up. The motivation behind this change was that `conservativeResizeLike()` cannot be called with `Eigen::MatrixBase<Derived>`.